service/grpc/codelingo/codelingo.pb.go

```diff
@@ -162,94 +162,79 @@ type PathsFromOffsetRequest struct {
 	End      int64  `protobuf:"varint,6,opt,name=end" json:"end,omitempty"`
 }
 
-func (m *PathsFromOffsetRequest) Reset()                    { *m = PathsFromOffsetRequest{} }
-func (m *PathsFromOffsetRequest) String() string            { return proto.CompactTextString(m) }
-func (*PathsFromOffsetRequest) ProtoMessage()               {}
-func (*PathsFromOffsetRequest) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{3} }
+func (m *QueryFromOffsetRequest) Reset()                    { *m = QueryFromOffsetRequest{} }
+func (m *QueryFromOffsetRequest) String() string            { return proto.CompactTextString(m) }
+func (*QueryFromOffsetRequest) ProtoMessage()               {}
+func (*QueryFromOffsetRequest) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{3} }
 
-func (m *PathsFromOffsetRequest) GetLang() string {
+func (m *QueryFromOffsetRequest) GetLang() string {
 	if m != nil {
 		return m.Lang
 	}
 	return ""
 }
 
-func (m *PathsFromOffsetRequest) GetDir() string {
+func (m *QueryFromOffsetRequest) GetDir() string {
 	if m != nil {
 		return m.Dir
 	}
 	return ""
 }
 
-func (m *PathsFromOffsetRequest) GetFilename() string {
+func (m *QueryFromOffsetRequest) GetFilename() string {
 	if m != nil {
 		return m.Filename
 	}
 	return ""
 }
 
-func (m *PathsFromOffsetRequest) GetSrc() string {
+func (m *QueryFromOffsetRequest) GetSrc() string {
 	if m != nil {
 		return m.Src
 	}
 	return ""
 }
 
-func (m *PathsFromOffsetRequest) GetStart() int64 {
+func (m *QueryFromOffsetRequest) GetStart() int64 {
 	if m != nil {
 		return m.Start
 	}
 	return 0
 }
 
-func (m *PathsFromOffsetRequest) GetEnd() int64 {
+func (m *QueryFromOffsetRequest) GetEnd() int64 {
 	if m != nil {
 		return m.End
 	}
 	return 0
 }
 
-type PathsFromOffsetReply struct {
-	Paths []*Path `protobuf:"bytes,1,rep,name=paths" json:"paths,omitempty"`
-}
-
-func (m *PathsFromOffsetReply) Reset()                    { *m = PathsFromOffsetReply{} }
-func (m *PathsFromOffsetReply) String() string            { return proto.CompactTextString(m) }
-func (*PathsFromOffsetReply) ProtoMessage()               {}
-func (*PathsFromOffsetReply) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{4} }
-
-func (m *PathsFromOffsetReply) GetPaths() []*Path {
-	if m != nil {
-		return m.Paths
-	}
-	return nil
-}
-
-type Path struct {
+type QueryFromOffsetReply struct {
 	Facts []*GenFact `protobuf:"bytes,1,rep,name=facts" json:"facts,omitempty"`
 }
 
-func (m *Path) Reset()                    { *m = Path{} }
-func (m *Path) String() string            { return proto.CompactTextString(m) }
-func (*Path) ProtoMessage()               {}
-func (*Path) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{5} }
+func (m *QueryFromOffsetReply) Reset()                    { *m = QueryFromOffsetReply{} }
+func (m *QueryFromOffsetReply) String() string            { return proto.CompactTextString(m) }
+func (*QueryFromOffsetReply) ProtoMessage()               {}
```

Comment:
exported method QueryFromOffsetReply.Descriptor should have comment or be unexported